### ISSUE TITLE
Installing fastwarc via `pip install` fails if compilation is required or requested

### ISSUE THREAD

**Issue by sebastian-nagel** [chatnoir-eu/chatnoir-resiliparse#16]
- applies to fastwarc 0.6.6 and 0.7.0 (0.6.5 successfully installed)
- seen on Ubuntu 20.04 and 21.04
- on amd64 with `pip3 install --no-binary fastwarc fastwarc`
- or on aarch64 with `pip3 install fastwarc` (no binaries provided for ARM CPUs)

The error message indicates that fastwarc is now too interconnected with resiliparse
```
  ERROR: Command errored out with exit status 1:
...  
  from resiliparse_common.string_util cimport str_to_lower, strip_str, strip_c_str
  ^
  ------------------------------------------------------------
  
  fastwarc/warc.pyx:32:0: 'resiliparse_common/string_util.pxd' not found
```

Building from a checkout of chatnoir-resiliparse via `pip3 wheel -e fastwarc` succeeds also on ARM-based systems.

**Comment by phoerious** [comment]
Fixed in 5b43781b714009d930719cec644920cb0890fe5c. Builds are underway.

**Comment by sebastian-nagel** [comment]
Thanks,  @phoerious: confirmed - installation with compilation works for 0.7.2
